apps/storybook/src/DataCurve.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import type { DataCurveProps } from '@h5web/lib';
+import { CurveType, DataCurve, useDomain, VisCanvas } from '@h5web/lib';
+import { assertDefined, mockValues } from '@h5web/shared';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { range } from 'lodash';
+
+import FillHeight from './decorators/FillHeight';
+
+const Template: Story<DataCurveProps> = (args) => {
+  const { abscissas, ordinates } = args;
+
+  const abscissaDomain = useDomain(abscissas);
+  const ordinateDomain = useDomain(ordinates);
+  assertDefined(abscissaDomain);
+  assertDefined(ordinateDomain);
+
+  return (
+    <VisCanvas
+      abscissaConfig={{ visDomain: abscissaDomain, showGrid: true }}
+      ordinateConfig={{ visDomain: ordinateDomain, showGrid: true }}
+    >
+      <DataCurve {...args} />
+    </VisCanvas>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  abscissas: range(0, mockValues.oneD.length),
+  ordinates: mockValues.oneD,
+  color: 'green',
+};
+Default.argTypes = {
+  showErrors: { control: false },
+};
+
+export const Glyphs = Template.bind({});
+Glyphs.args = {
+  abscissas: range(0, mockValues.oneD.length),
+  ordinates: mockValues.oneD,
+  color: 'black',
+  curveType: CurveType.GlyphsOnly,
+};
+Glyphs.argTypes = {
+  showErrors: { control: false },
+};
```

Comment:
It would perhaps make more sense to set this in an `argTypes` field in the `export` below and activate the control only for `WithErrors`.

Reply:
I tried, but it didn't work for some reason 🤷

Reply:
Nvm, go ahead then :+1: